LocalPlayer a nil value when trying to write version info.

@KajTorvaldGrey messaged me with this error on Steam. This is for when the system checks for a new version here on github.

Link to location https://github.com/lzinga/TTTWeightedTraitorSelection/blob/master/tttweightsystem/lua/autorun/weightsystem_autorun.lua#L115

```
[ERROR] lua/autorun/weightsystem_autorun.lua:115: attempt to call global 'LocalPlayer' (a nil value)
1. onsuccess - lua/autorun/weightsystem_autorun.lua:115
2. unknown - lua/includes/modules/http.lua:35
```

Last time I was working on this addon it was working, so maybe something changed with LocalPlayer method or just need to change it so it has a nil check or loop over all players in the game and write it instead of using LocalPlayer.

--- a/tttweightsystem/lua/autorun/weightsystem_autorun.lua
+++ b/tttweightsystem/lua/autorun/weightsystem_autorun.lua
@@ -75,45 +75,46 @@ else
 end
 
 
-
-hook.Add("InitPostEntity", "TTTWS_updatecheck", function()
-		http.Fetch("https://raw.githubusercontent.com/lzinga/TTTWeightedTraitorSelection/master/tttweightsystem/lua/autorun/weightsystem_autorun.lua", function(body)
-			local str = string.match(body, "[^\n]+")
-			local ver = str:sub(18, -3)
-			local major, minor, patch = ver:match("(%d+)%.(%d+)%.(%d+)")
-			major = tonumber(major) or 0
-			minor = tonumber(minor) or 0
-			patch = tonumber(patch) or 0
-			local curmajor, curminor, curpatch = version:match("(%d+)%.(%d+)%.(%d+)")
-			curmajor = tonumber(curmajor) or 0
-			curminor = tonumber(curminor) or 0
-			curpatch = tonumber(curpatch) or 0
-			local msg
-			if major > curmajor then
-				msg = [[
-A new major version of TTT Weight System is
-available (%%%%%%%%).
-]]
-			elseif minor > curminor then
-				msg = [[
-A new minor version of TTT Weight System is
-available (%%%%%%%%).
-]]
-			elseif patch > curpatch then
-				msg = [[
-A new patch is available for
-TTT Weight System (%%%%%%%%).
-]]
-			else
-				msg = [[
-[TTT Weight System] is up to date (%%%%%%%%).
-]]
-			end
-			if msg then
-				msg = string.gsub(msg, "%%+", ver)
-				for w in string.gmatch(msg, "[^\n]+") do
-					LocalPlayer():PrintMessage(HUD_PRINTTALK , w .. string.rep(" ", 32 - #w))
+if CLIENT then
+	hook.Add("InitPostEntity", "TTTWS_updatecheck", function()
+			http.Fetch("https://raw.githubusercontent.com/lzinga/TTTWeightedTraitorSelection/master/tttweightsystem/lua/autorun/weightsystem_autorun.lua", function(body)
+				local str = string.match(body, "[^\n]+")
+				local ver = str:sub(18, -3)
+				local major, minor, patch = ver:match("(%d+)%.(%d+)%.(%d+)")
+				major = tonumber(major) or 0
+				minor = tonumber(minor) or 0
+				patch = tonumber(patch) or 0
+				local curmajor, curminor, curpatch = version:match("(%d+)%.(%d+)%.(%d+)")
+				curmajor = tonumber(curmajor) or 0
+				curminor = tonumber(curminor) or 0
+				curpatch = tonumber(curpatch) or 0
+				local msg
+				if major > curmajor then
+					msg = [[
+	A new major version of TTT Weight System is
+	available (%%%%%%%%).
+	]]
+				elseif minor > curminor then
+					msg = [[
+	A new minor version of TTT Weight System is
+	available (%%%%%%%%).
+	]]
+				elseif patch > curpatch then
+					msg = [[
+	A new patch is available for
+	TTT Weight System (%%%%%%%%).
+	]]
+				else
+					msg = [[
+	[TTT Weight System] is up to date (%%%%%%%%).
+	]]
 				end
-			end
-		end)
-end)
+				if msg then
+					msg = string.gsub(msg, "%%+", ver)
+					for w in string.gmatch(msg, "[^\n]+") do
+						LocalPlayer():PrintMessage(HUD_PRINTTALK , w .. string.rep(" ", 32 - #w))
+					end
+				end
+			end)
+	end)
+end

